### PR TITLE
[ENGG-2280] feat: display bluecore events

### PR DESCRIPTION
**Pull request by rohanmathur91**

### CODE REVIEW

File: browser-extension/common/package.json [hunk 1]
```diff
@@ -56,9 +56,11 @@
     "@devtools-ds/object-inspector": "^1.2.1",
     "@devtools-ds/table": "^1.2.1",
     "@devtools-ds/themes": "^1.2.1",
+    "@types/lodash": "^4.17.7",
```
Comment (nafees87n) [inline browser-extension/common/package.json:59]: Move types to devDependencies

File: browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEvent/VendorEvent.tsx
Comment (nafees87n) [inline browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEvent/VendorEvent.tsx:1]: Can we name it something like `VenderEventComponent` or something else? Currently it does not look like a component

Comment (rohanmathur91) [reply browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEvent/VendorEvent.tsx:1]: Its already inside `/components/` so naming it `VenderEventComponent` will go against convention, so keeping it as it is for now.

Comment (nafees87n) [reply browser-extension/common/src/devtools/containers/analytics-inspector/components/VendorEvent/VendorEvent.tsx:1]: What about `VendorEventDisplay` or something that the component represents? `VendorEvent` is a noun specfic to events, we also have state variable names as `vendorEvents`, There will be types related to `vendorEvent`. Having a component named such will create confusion.

**Review by nafees87n — APPROVED** [review]

**Pull request merged**